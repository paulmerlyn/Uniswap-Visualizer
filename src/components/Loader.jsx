import Spinner from "react-bootstrap/Spinner"
import Stack from "react-bootstrap/Stack"

const Loader = () => {
  return (
    <Stack className="d-flex align-items-center text-center mx-auto">
      <div style={{ height: 100 }}></div>
      <Spinner className="d-flex align-items-center text-center" animation="border" role="status">
        <span className="visually-hidden">Loading...</span>
      </Spinner>
    </Stack>
  )
}

export default Loader
