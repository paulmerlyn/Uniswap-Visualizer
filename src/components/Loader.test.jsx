import renderer from "react-test-renderer"
import Loader from "./Loader"

it("renders correctly", () => {
  const component = renderer.create(<Loader />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
