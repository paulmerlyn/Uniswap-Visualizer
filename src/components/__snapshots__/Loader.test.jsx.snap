// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="d-flex align-items-center text-center mx-auto vstack"
>
  <div
    style={
      Object {
        "height": 100,
      }
    }
  />
  <div
    className="d-flex align-items-center text-center spinner-border"
    role="status"
  >
    <span
      className="visually-hidden"
    >
      Loading...
    </span>
  </div>
</div>
`;
